examples/data_preprocessing/[WIP]_chexpert_dataset/Prepare_CheXpert_data.ipynb

```diff
@@ -0,0 +1,478 @@
+    "https://github.com/gaetandi/cheXpert/blob/master/cheXpert_final.ipynb and https://github.com/Stomper10/CheXpert/blob/master/CheXpert_DenseNet121_FL.ipynb"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "transform_list = transforms.Compose([\n",
+    "    transforms.Resize((320, 320)),\n",
+    "    transforms.ToTensor(),\n",
+    "    transforms.Normalize([0.485, 0.456, 0.406], [0.229, 0.224, 0.225]) #normalization from ImageNet\n",
+    "    ])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The following class will be used to transform the images and prepare the dataset for PyTorch's DataLoader. Moreover, the uncertainty labels will be handled according to either the \"U-Ones\", \"U-Zeroes\", \"U-Ignore\" or \"U-MultiClass\" approach as they were described in the origianl CheXpert paper by Irvin et al. (2019). You can specify how the uncertainty labels for each individual pathology will be handled by providing a list of the names of the pathologies to the arguments to_ones, to_zeros and to_ignore. to_ignore changes the uncertainty label (-1.0) to nan. Not specifying any pathology in any of the arguments to_ones, to_zeros or to_ignore will train the model with the \"U-MultiClass\" approach. The \"Support Device\" label as well as the \"No Finding\" label is dropped for all instances, since support devices do not count as pathologies and the \"No Finding\" label only depends on the labels for the remaining 12 pathologies. The \"blank\" labels, indicating that an observation was not mentioned in the corresponding report, will be treated as \"uncertain\". The validation set can be loaded by setting the subset input of the class to \"valid\".\n",
+    "\n",
+    "The structure of the class was inspired by the github submissions credited above."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class prepare_CheXpert(Dataset):\n",
+    "    \n",
+    "    def __init__(self, path, subset = \"train\", to_ones = None, to_zeros = None, to_ignore = None, transform_sequence = None):\n",
+    "        \n",
+    "        \"\"\"\n",
+    "        path: path to the folder where train.csv and valid.csv are stored\n",
+    "        subset: either \"train\" to load the train.csv or \"valid\" to load valid.csv\n",
+    "        to_ones: list of pathologies for which uncertainty labels should be replaced by 1\n",
+    "        to_zeros: list of pathologies for which uncertainty labels should be replaced by 0\n",
+    "        to_ignore: list of pathologies for which uncertainty labels should be ignored (label will be turned to nan)\n",
+    "        transform_sequence: sequence used to transform the images\n",
+    "        \"\"\"\n",
+    "        \n",
+    "        #read dataset\n",
+    "        if subset == \"train\":\n",
```

Comment:
normally no real work should be done in `def __init__()` but rather just the initialization of self parameters

then:
- in `def __init__()` there would be only lines 310-313
- the rest of the work should be split in different functions, so that 
     - one function that would be called in cell 11 (e.g. `def process_chexpert_dataset`)
     - other functions that would be called from it